pom.xml

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.mskcc.cmo</groupId>
+  <artifactId>irt-publisher</artifactId>
+  <name>CMO IRT Publisher Master</name>
+  <version>0.0.1-SNAPSHOT</version>
+  <description>IRT Sample Status Publisher project for Spring Boot</description>
+  <packaging>jar</packaging>
+
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>2.3.3.RELEASE</version>
+  </parent>
+
+  <properties>
+    <java.version>1.8</java.version>
+    <maven.compiler.version>1.8</maven.compiler.version>
+    <spring.boot.version>2.3.3.RELEASE</spring.boot.version>
+    <spring.version>5.2.8.RELEASE</spring.version>
+    <jackson.version>2.11.2</jackson.version>
+    <!-- metadb messaging -->
+    <cmo_metadb_messaging_java.group>com.github.mskcc</cmo_metadb_messaging_java.group>
+    <cmo_metadb_messaging_java.version>0cc7bcc0cafe00421acc79171b6bf2f03660d854</cmo_metadb_messaging_java.version>
```

Comment:
updated the messaging library: 

new commit hash: `38bb705c25ff8380d78df7a45698d3f84086de17`

and new property to add 
```
nats.filter_subject=METADB.*
```

Reply:
Done and done (I have to check if the app-props file in configuration has this property).